app/assets/stylesheets/styles.scss

```diff
@@ -414,4 +414,9 @@ a > img {
 
 .bg-light-blue {
   background-color :#8ecbeb;
-}
\ No newline at end of file
+}
+
+.manual-sync-button {
+  height: 300px;
```

Comment:
I'll accept it https://media1.tenor.com/m/U-poF-2UW6AAAAAd/i-am-a-generous-god-xerxes.gif